CHANGES.md

```diff
@@ -10,6 +10,11 @@
 * Fix `Ptime.truncate` to always truncate down. Thanks
   to David Kaloper Meršinjak for the report & fix.
 
+v0.8.6 2021-11-26 Jonah Beckford
+--------------------------------
+
+* Allow compiling with MSVC compiler
+
```

Comment:
Could you please simply add this to the list above with your name.

Reply:
Oops; didn't understand the top of CHANGES.md. Fixed.